### PR TITLE
Priyanka_Assignment_30

### PR DESCRIPTION
**Pull request by PriyankaKOct2020**

### CODE REVIEW

File: src/priyanka/Assignment30/FindMissing.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+/*Assignment 30 : 29th Nov 2020 
+Program 1:  Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but exactly one number is missing, you need to write a Java program to find that missing number in an array.
+*/
+
+public class FindMissing {
+	void findMissingNumber(int[] arr1, int[] arr2) {
+
+		for (int i = 0; i < arr1.length; i++) {
```
Comment (harshvegada) [inline src/priyanka/Assignment30/FindMissing.java:13]: You have to find missing number from single array

**Review by harshvegada — COMMENTED** [review]
pls check requirement again & acknowledge me

File: src/priyanka/Assignment30/FindMissing.java (hunk 1 above)
Comment (PriyankaKOct2020) [reply src/priyanka/Assignment30/FindMissing.java]: Hi Harsh,  

updated the logic for missing number

**Review by harshvegada — APPROVED** [review]
Code looks good to me!
Good to merge

**Pull request merged**